docs/tutorials/ionq/binary_paintshop.ipynb

```diff
@@ -0,0 +1,421 @@
+{
```

Comment:
<p>If cirq does not install this won't import cirq or ionq.  Suggest</p><pre class="ql-syntax" spellcheck="false">try:
&nbsp; &nbsp; import cirq
except ImportError:
&nbsp; &nbsp; print("installing cirq...")
&nbsp; &nbsp; !pip install --quiet cirq
&nbsp; &nbsp; print("installed cirq.")
import numpy as np
import cirq
import cirq.ionq as ionq
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>